tsconfig.json

```diff
@@ -32,107 +32,113 @@
 	"skipDefaultLibCheck": true,
 	"paths": {
 	  "@webb-dapp/apps/*": [
-		"apps/src/*"
+			"apps/src/*"
```

Comment:
Might want to fix formatting?

Reply:
I'm not sure why the git diff is making it look worse - when I check the source code in my editor as well as the code at the particular commit, these changes are actually fixing formatting.